packages/flutter/test/material/scaffold_test.dart

```diff
@@ -568,6 +568,15 @@ void main() {
       );
     }
 
+    PageRoute<void> pageRouteBuilder() {
+      return PageRouteBuilder(
```

Comment:
It looks like the analyzer is failing here where `PageRouteBuilder` is missing a type specifier.

Reply:
Oops, fixed. Thanks for the finding!